assets/airports/loww.json

```diff
@@ -386,448 +362,339 @@
                 "16": [["WW267", "S205-"], "WW160", "WW174"],
-                "34": [["WW165", "A15|S205-"], "WW371"]
+                "29": [["WW293", "A10+|S205-"], "WW230", "WW266"],
+                "34": [["WW165", "A15+|S205-"], "WW371"]
             },
             "exitPoints": {
                 "LEDVA": ["LEDVA"]
             },
-            "draw": [["WW170", "LEDVA"], ["WW266", "LEDVA"], ["WW160", "WW174", "LEDVA"], ["WW371", "LEDVA"]]
+            "draw": [
+                ["WW170", "LEDVA"],
+                ["WW266", "LEDVA"],
+                ["WW160", "WW174", "LEDVA"],
+                ["WW371", "LEDVA"]
+            ]
         },
-        "LUGIM": {
-            "icao": "LUGIM",
-            "name": "Lugim",
-            "suffix": {"11":"1A", "29":"1C", "16":"3B", "34":"2D"},
+        "LUGEM": {
+            "icao": "LUGEM",
+            "name": "Lugem",
+            "suffix": {"11":"1A", "16":"3B", "29":"1C", "34":"2D"},
             "rwy": {
                 "11": [["WW100", "S205-"], "WW101", ["WW274", "A40+"]],
-                "29": [["WW293", "A10|S205-"], "WW232", ["WW231", "A40+"]],
                 "16": [["WW269", "S205-"], "WW265", ["WW379", "A35+"], "WW381"],
+                "29": [["WW293", "A10+|S205-"], "WW232", ["WW231", "A40+"]],
                 "34": [["WW304", "A13+|S205-"], "WW293", "WW232", ["WW231", "A40+"]]
             },
             "exitPoints": {
-                "LUGIM": ["WW177", "LUGIM"]
+                "LUGEM": ["LUGEM"]
             },
-            "draw": [["WW100", "WW101", "WW274", "WW177"], ["WW269", "WW265", "WW379", "WW381", "WW177"], ["WW304", "WW293", "WW232", "WW231", "WW177", "LUGIM"]]
+            "draw": [
+                ["WW100", "WW101", "WW274"],
+                ["WW269", "WW265", "WW379", "WW381"],
+                ["WW304", "WW293", "WW232", "WW231", "LUGEM"]
+            ]
         },
-        "MOTIX": {
-            "icao": "MOTIX",
-            "name": "Motix",
-            "suffix": {"11":"1A", "29":"1C", "16":"3B", "34":"1D"},
+        "MEDIX": {
+            "icao": "MEDIX",
+            "name": "MEDIX",
+            "suffix": {"11":"1A", "16":"3B", "29":"1C", "34":"1D"},
             "rwy": {
                 "11": [["WW100", "S205-"], "WW101", ["WW273", "A40+"]],
-                "29": [["WW293", "A10|S205-"], "WW232", ["WW231", "A40+"]],
                 "16": [["WW269", "S205-"], "WW265", "WW379", "WW382"],
+                "29": [["WW293", "A10+|S205-"], "WW232", ["WW231", "A40+"]],
                 "34": [["WW304", "A13+|S205-"], "WW293", ["WW230", "A40+"]]
             },
             "exitPoints": {
-                "MOTIX": ["WW176", "MOTIX"]
+                "MEDIX": ["MEDIX"]
             },
-            "draw": [["WW101", "WW273", "WW176"], ["WW231", "WW176"], ["WW379", "WW382", "WW176"], ["WW230", "WW176", "MOTIX"]]
+            "draw": [
+                ["WW101", "WW273"],
+                ["WW231"],
```

Comment:
This only includes a single point, and thus cannot be drawn. Each array within this 2D array is a series of points between which we are to draw lines. So we need at least two points to draw a line between.